SAM.Picker/GamePicker.cs

```diff
@@ -270,7 +262,6 @@ private void AddGameToLogoQueue(GameInfo info)
             {
                 this._LogosAttempted.Add(logo);
                 this._LogoQueue.Enqueue(info);
-                this.DownloadNextLogo();
```

Comment:
Triggered by code that calls `AddGameToLogoQueue` (once, not for every app) anyway.